packages/components/src/internal/components/buttons/ToggleButtons.tsx

```diff
@@ -1,23 +1,26 @@
-import React from 'react';
+import React, { PureComponent, ReactNode } from 'react';
 import { Button } from 'react-bootstrap';
 
 interface Props {
     first: string;
     second: string;
     active?: string;
     onClick: (selected: string) => void;
+    className?: string;
 }
 
-export class ToggleButtons extends React.Component<Props, any> {
-    render() {
-        const { first, second, onClick, active } = this.props;
+export class ToggleButtons extends PureComponent<Props> {
+    render(): ReactNode {
+        const { first, second, onClick, active, className } = this.props;
+        const firstActive = active === first;
+        const secondActive = active === second;
 
         return (
-            <div className="btn-group">
-                <Button active={active === first} bsSize="small" onClick={() => onClick(first)}>
+            <div className={'btn-group' + (className ? ' ' + className : '')}>
+                <Button bsStyle={firstActive ? 'primary' : 'default'} bsSize="small" onClick={() => onClick(first)}>
                     {first}
                 </Button>
-                <Button active={active === second} bsSize="small" onClick={() => onClick(second)}>
+                <Button bsStyle={secondActive ? 'primary' : 'default'} bsSize="small" onClick={() => onClick(second)}>
```

Comment:
I'm curious why this change was necessary and whether we'll see a change in style elsewhere because of this.  What was the default styling if the button was just "active"?

Reply:
A `<Button>` with active=true will be in the "pressed" looking state, which helps distinquish it here from the non-active size of the ToggleButton. The mockups for this story has the active/selected button in the group using the primary LK theme color so I thought that'd be a good default of this component. Yes this change will affect all usages of it...but there aren't any usages in the apps or otherwise besides the storybook ones.

Reply:
Gotcha.  Thanks for the info.